docs/networks.md

```diff
@@ -1,27 +1,173 @@
 # Using Networks
 
+## Network creation
+
 With the release of Docker 1.9 you can now manage custom networks.
 
 
-Here you can see how to create a network named ```network1``` using the ```bridge``` driver
+Here you can see how to create a network named `network1` using
+the `bridge` driver
 
 ```python
 docker_client.create_network("network1", driver="bridge")
 ```
 
-You can also create more advanced networks with custom IPAM configurations. For example, 
-setting the subnet to ```192.168.52.0/24``` and gateway to ```192.168.52.254```
+You can also create more advanced networks with custom IPAM configurations.
+For example, setting the subnet to `192.168.52.0/24` and gateway address
+to `192.168.52.254`
 
 ```python
-
-ipam_config = docker.utils.create_ipam_config(subnet='192.168.52.0/24', gateway='192.168.52.254')
+ipam_pool = docker.utils.create_ipam_pool(
+    subnet='192.168.52.0/24',
+    gateway='192.168.52.254'
+)
+ipam_config = docker.utils.create_ipam_config(
+    pool_configs=[ipam_pool]
+)
 
 docker_client.create_network("network1", driver="bridge", ipam=ipam_config)
 ```
 
-With Docker 1.10 you can now also create internal networks
+By default, when you connect a container to an overlay network, Docker also
+connects a bridge network to it to provide external connectivity. If you want
+to create an externally isolated overlay network, with Docker 1.10 you can
+create an internal network.
 
 ```python
 
 docker_client.create_network("network1", driver="bridge", internal=True)
 ```
+
+## Container network configuration
+
+In order to specify which network(s) a container will be connected to and
+additional configuration, use the `networking_config` parameter in
+`Client.create_container`
```

Comment:
This is a little misleading. When creating a container, you can only connect it to one network, using the `network_mode` argument to `create_host_config`. You can only specify that one network's config in the `networking_config` dict (so the example below is fine).

To connect a container to multiple networks, you have to first create it with one of the networks specified, then call `connect_container_to_network` once for each remaining network, with the relevant endpoint config. We should probably be explicit about this.